test_autofit/text/test_text_util.py

```diff
@@ -37,7 +37,7 @@ def test__results_to_file(samples):
         samples=samples,
     )
 
-    assert  "Maximum Log Likelihood                                                                    1.00000000\n" in result_info
+    assert  "Maximum Log Likelihood                                                          1.00000000\n" in result_info

```

Comment:
I wonder if it's worth making this test dynamic

Reply:
Don't think so, the config is a tesrt specific one. Worry that dynamic might miss changes to source code.